app/views/conference/_schedule.html.erb

```diff
@@ -13,51 +13,39 @@
         <div class="tab-pane <%= 'active' if @dates.first == date %>" id ="<%= date %>" >
           <table class="table table-bordered" id="schedule" width="100%">
             <tr>
-              <th>Time</th>
-              <% @rooms.each do |room| %>
-                <th><%= room.name %></th>
+              <th>Rooms</th>
+              <%# The conference starts at 9 and ends at 18:45 %>
+              <% intervals = (19 - 9) * 4 %>
```

Comment:
I am iterating over the number of hours. I need the intervals to set the width of each column, although I could calculate the width directly. I am changing that and moving it to the controller.